.github/workflows/test-batch-processing.yml

```diff
@@ -22,6 +22,7 @@ jobs:
   batch_processing:
     name: Test batch_processing.sh
     strategy:
+      fail-fast: false
```

Comment:
I'm surprised we hadn't added this before -- now at least runs will continue across the matrix even if one fails.